### PR TITLE
Skit monitoring pr pipeline support

### PR DESCRIPTION
**Pull request by TBolton2000**
Code complete. Merge with https://github.com/IBM/devex-skit-monitor-toolchain/pull/35

### CODE REVIEW

**Comment by TBolton2000** [comment]
Code changes:

- Added deployment assets for helm for PR for each skit
- scripts/copy_assets.sh now directly uses newly added SKIT_NAME environment variable
- scripts/deploy_cf.sh is passed the actual app name, so we don't build it here
- scripts/skit_registration.sh can also use newly added SKIT_NAME
- scripts/verification_experience_test.sh added flag for slack message and PagerDuty, added a flag for REGISTER_SKIT

File: scripts/skit_alert_and_init_registration.sh [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+  source <(curl -sSL "$DEVX_SKIT_ASSETS_GIT_URL_RAW/scripts/skit_registration.sh")
+  echo "Beginning skit registration..."
+  register_skit
+  if [ $REG_EXIT != 0 ]; then
+    msg="Skit registration failed. Check the starter-kit-registration Tekton pipeline logs under DevOps Toolchains for details."
+    fail_msg=":spinning-siren: $msg"
+    source <(curl -sSL "$DEVX_SKIT_ASSETS_GIT_URL_RAW/scripts/slack_message.sh") "$fail_msg"
+    exit 1
+  fi
+fi
```
Comment (nfstein) [inline scripts/skit_alert_and_init_registration.sh]: We can delete this file too, right?

**Comment by TBolton2000** [comment]
TC: https://cloud.ibm.com/devops/toolchains/5e4a73bf-ac3d-4175-8abe-73ddae78110b?env_id=ibm:yp:us-south

File: deployment-assets/natural-language-understanding-code-pattern/helm/naturallanguageunderstandingmonitoredhelmpr/Chart.yaml [hunk 2]
```diff
@@ -0,0 +1,4 @@
+apiVersion: v1
+description: A Helm chart for Kubernetes
+name: naturallanguageunderstandingmonitoredhelmpr
```
Comment (gee4vee) [inline deployment-assets/natural-language-understanding-code-pattern/helm/naturallanguageunderstandingmonitoredhelmpr/Chart.yaml:3]: Do these long names cause issues at deploy time? I seem to recall Helm has limits.

**Review by gee4vee — APPROVED** [review]
As long as the long name in the NLU helm chart isn't a problem, this looks good.

File: deployment-assets/natural-language-understanding-code-pattern/helm/naturallanguageunderstandingmonitoredhelmpr/Chart.yaml (hunk 2 above)
Comment (nfstein) [reply deployment-assets/natural-language-understanding-code-pattern/helm/naturallanguageunderstandingmonitoredhelmpr/Chart.yaml:3]: Shouldnt, length requirement is 63 char I think

**Review by nfstein — APPROVED** [review]

**Pull request merged**